fetch.bs

```diff
@@ -589,6 +589,9 @@ user-agent-defined <a for=header>value</a> for the
 
  <li><p>A <dfn export for=body id=concept-body-total-bytes>total bytes</dfn> (an
  integer), initially 0.
+
+ <li><p>A <dfn export for=body id=concept-body-replayable>replayable flag</dfn> (a boolean),
+ initially false.
```

Comment:
Also, I think we want to call this cloneable maybe? That is what the operation we want to fail is called.